packages/ui-webc/src/components/text-area/text-area.css

```diff
@@ -0,0 +1,21 @@
+:host {
+  display: inline-flex;
+}
+
+.flexbox{
+  display: inline-flex;
+  flex-direction: column;
+  padding: 0.5rem;
+}
+
+.textarea{
+  resize: none;
+  height: var(--spacing-24);
+  width: var(--spacing-50);
```

Comment:
I think we should leave out the width and height and let the consumer control this as it will most likely depend on their layout. Maybe instead we can expose the [rows attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/textarea#rows) as a prop and set the default to something a bit higher, say 3?